Gordon360/Controllers/ProfilesController.cs

```diff
@@ -89,16 +89,32 @@ public ProfilesController(IProfileService profileService, IAccountService accoun
                 faculty = _faculty == null ? null : (PublicFacultyStaffProfileViewModel)_faculty;
                 alumni = _alumni == null ? null : (PublicAlumniProfileViewModel)_alumni;
             }
+            else if (viewerGroups.Contains(AuthGroup.Alumni) && viewerGroups.Contains(AuthGroup.Student))
+            {
+                // Ask Chris about what group (Alumni + Student) can see
+                student = _student == null ? null : (PublicStudentProfileViewModel)_student;
+                faculty = _faculty == null ? null : (PublicFacultyStaffProfileViewModel)_faculty;
+                alumni = _alumni == null ? null : (PublicAlumniProfileViewModel)_alumni;
+            }
+            else if (viewerGroups.Contains(AuthGroup.Alumni))
+            {
+                /* student = _student == null ? null : (PublicStudentProfileViewModel)_student;
+                faculty = _faculty == null ? null : (PublicFacultyStaffProfileViewModel)_faculty; */
+                // Ask Chris about what group Alumni can see
```

Comment:
```suggestion
                // Ask Chris Carlson about what group Alumni can see
```

Reply:
Oh I forgot to write the last name, sorry! Will add it now